.github/workflows/ci.yaml

```diff
@@ -30,3 +30,12 @@ jobs:
       run: |
         pip install  .
         make test
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: pip install .[dev]
+    - name: Lint
+      run: pylint --rcfile=pylintrc src
```

Comment:
Hmm, why not `make lint` ?

Reply:
I'll try that again, but I think it was raising an error because it wasn't being called in the project directory

Reply:
Looks like I misread the error before and it works fine :)